src/AccessibilityInsights.Core/Bases/A11yElement.cs

```diff
@@ -336,6 +423,9 @@ public T GetPlatformPropertyValue<T>(int propertyId)
             return property.Value;
         }
 
+        /// <summary>
+        /// the index of an item within a set
```

Comment:
>set [](start = 42, length = 3)

Seems like "set" would be clearer with some context. I'm assuming that it refers to the siblings in the same container? Same comment applies to SizeOfSet

Reply:
I don't think we can be as specific as siblings. I also think we can't explain everything in our code documentation. I think we should provide a practical hint and rely on external docs to provide more info.